src/main/java/com/artipie/asto/memory/InMemoryStorage.java

```diff
@@ -85,7 +85,18 @@
 
     @Override
     public CompletableFuture<Void> move(final Key source, final Key destination) {
-        throw new UnsupportedOperationException();
+        return CompletableFuture.runAsync(
+            () -> {
+                final byte[] bytes = this.data.get(source.string());
```

Comment:
@olegmoz instead of calling `get` and doing a null check, we should do a `containsKey` check. Then we get the data or throw an error depending on the result.

Reply:
@carlosmiranda fixed this